### PR TITLE
Implement API

### PR DESCRIPTION
**Pull request by Munsa1**
In the following milestone i have: 

- Created a new game using an API.

- Implemented the "Refresh" button (receiving data from the API and parsing the JSON).

- Implemented the form "Submit" button (sending data to the API).

- Used async and await JavaScript features to consume the API.

No styling is required.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,68 +1,50 @@
...
+```cmd
 npm install
 ```
 
 ### Usage
 
-```bash
+```cmd
 npm start
 ```
```
Comment (carlos-ssh) [inline README.md:41]: Hi your form it's working well but i can't see the json form the API printed on your recent scores.
Kindly add your resources from your api requests.
Screen Shot 2021-09-04 at 15 02 14

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @Munsa1 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Munsa1) [reply README.md:41]: It's been handled in the util.js file. thanks for the feedback👌👌

File: README.md [hunk 2]
```diff
@@ -1,68 +1,50 @@
...
 
-👤 **Babakolo Usman Suleiman**
+👤 **Munsa Mibenge**
 
-- GitHub: [@usmansbk](https://github.com/usmansbk)
-- LinkedIn: [LinkedIn](https://www.linkedin.com/in/usman-suleiman-82b444140/)
+- GitHub: [@munsa1](https://github.com/munsa1)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/munsa-mibenge-a35736205/)
 
 ## 🤝 Contributing
```
Comment (carlos-ssh) [inline README.md:50]: Try to add the Id into your API link that it's described into your [API documentation](https://www.notion.so/Leaderboard-API-service-24c0c3c116974ac49488d4eb0267ade3).
It's needed to get the json from the url and display it in your recent scores list.

I hope that this clue from the documentation going to be helpful for you and try to implement it in your util.js file:
Screen Shot 2021-09-04 at 15 53 10

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @Munsa1 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by carlos-ssh — APPROVED** [review]
Hi @Munsa1 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**